gnomad_qc/v4/annotations/generate_freq_genomes.py

```diff
@@ -839,15 +871,18 @@ def compute_an_by_group_membership(
             sex_karyotype=vmt.meta.sex_imputation.sex_karyotype,
             group_membership=group_membership_ht[vmt.col_key].group_membership,
         )
-        .select_entries("AD", "DP", "GT", "GQ")
+        .select_entries("LA", "LAD", "DP", "LGT", "GQ")
```

Comment:
why changed to "LAD" and "LGT"?

Reply:
I moved the split to after this, so we need to select the unsplit annotations